CRISPResso2/CRISPRessoShared.py

```diff
@@ -564,6 +565,11 @@ def check_output_folder(output_folder):
 # Thanks https://gist.github.com/simonw/7000493 for this idea
 class CRISPRessoJSONEncoder(json.JSONEncoder):
     def default(self, obj):
+        if isinstance(obj, ResultsSlotsDict):
```

Comment:
Could we have this be `CRISPRessoCOREResources.ResultsSlotsDict` so that we can remove the import on line 30?